core/src/templates/components/local-footer/local-footer.twig

```diff
@@ -0,0 +1,206 @@
+      {% if social_links is not empty%}
+      <ul class="su-local-footer__social-links">
+        {% if social_links is iterable %}
+          {{ list_macro.link_list(social_links) }}
+        {% else %}
+          {{ social_links }}
+        {% endif %}
+      </ul>
+      {% endif %}
+      {% endblock %}
+      {# Use this var if you would like to place your own content in to this cell. #}
+      {{ cell1 }}
+    </div>
+    {# Content Cell 2 - For primary and secondary link lists. #}
+    <div class="su-local-footer__cell2">
+      {# Twig block for you to extend with #}
+      {% block contentcell2 %}
+      {#
+        Primary Links - left or top list.
+      #}
+      {% if primary_links is not empty%}
+      <nav aria-label="footer primary nav">
+        {% if primary_links_header is not empty %}
+        <h2 class="su-local-footer__list-heading">{{ primary_links_header|default("Primary Links") }}</h2>
+        {% endif %}
+        <ul class="su-local-footer__primary-links">
+          {# Stylized links. See local-footer.json for structure for macro. #}
+          {% if primary_links is iterable %}
+            {{ list_macro.link_list(primary_links) }}
+          {% else %}
+            {{ primary_links }}
+          {% endif %}
+        </ul>
+      </nav>
+      {% endif %}
+      {#
+        Secondary Links - Right or bottom list.
+      #}
+      {% if secondary_links is not empty%}
+      <nav aria-label="footer secondary nav">
+        {% if secondary_links_header is not empty %}
+        <h2 class="su-local-footer__list-heading">{{ secondary_links_header|default("Secondary Links") }}</h2>
+        {% endif %}
+        <ul class="su-local-footer__secondary-links">
+          {# Stylized links. See local-footer.json for structure for macro. #}
+          {% if secondary_links is iterable %}
+            {{ list_macro.link_list(secondary_links) }}
+          {% else %}
+            {{ secondary_links }}
+          {% endif %}
+        </ul>
+      </nav>
+      {% endif %}
+      {% endblock %}
+      {# If you want to provide all of our own markup use the cell2 var. #}
+      {{ cell2 }}
+    </div>
+    {#
+      Cell 3 - For the signup form.
+    #}
+    <div class="su-local-footer__cell3">
+      {# Twig block for you to extend with #}
+      {% block contentcell3 %}
+      {#
+        The signup form is optional. If it doesn't meet your content needs
+        use the cell3 variable instead.
+      #}
+      {% if signup_form_action is not empty %}
+      {# Add any additional markup above the form with this variable. #}
+      {{ signup_form_content }}
+      <form class="su-local-footer__form" action="{{ signup_form_action }}" method="{{ signup_form_method|default('post') }}">
+        <input type="email" id="{{ signup_form_field_email }}" name="{{ signup_form_field_email }}" placeholder="{{ signup_form_field_email_placeholder|default('email address') }}" {{ signup_form_field_email_required|default('required') }} />
```

Comment:
Could we add a screen reader-only label for this input field? It's an a11y thing. We did that for the site search component also.